docs/source/upgrade_notes.rst

```diff
@@ -47,6 +47,16 @@ Upgrade Notes
   workflow definition contains a task that is named ``continue``.
 * When installing packs from Exchange index ``st2 pack install <pack_name>`` will now download latest
   release from the remote repository, instead of using latest available git commit from master as before.
+* When upgrading an installation with the Open Source LDAP auth backend ``st2_auth_backend_ldap``
```

Comment:
> Open Source LDAP auth backend

Considering EWC LDAP is now open-sourced, this needs proper re-wording.

Reply:
Additionally, keeping in mind it's a `v3.2` upgrade notes, I guess you want to change the stable docs, eg `3.2` branch of st2docs instead of master. Otherwise this change will go live in "stable" docs version when we release v3.3.

Reply:
I would refer to it as the "Community LDAP auth backend" to differentiate it from the "(formerly) Enterprise LDAP auth backend". Don't mention open source at all, since that doesn't differentiate the two anymore.